pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -118,7 +118,13 @@ def work(self):
                                     seconds_since_epoch)))
             elif spin_result == 4:
                 logger.log("Inventory is full, switching to catch mode...", 'red')
-                self.config.mode = 'poke'
+                #self.config.mode = 'poke'
+                self.api.get_player()
+                response_dict_player = self.api.call()
+                player = response_dict_player['responses']['GET_PLAYER']['player_data']
+                items_count = self.bot.get_inventory_count('item')
+                items_stock = self.bot.current_inventory()
+                self.bot.check_inventory_space(items_count, player['max_item_storage'], items_stock.copy())
```

Comment:
This shouldn't be done here, it should be added here: https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/pokemongo_bot/__init__.py#L45

That means it will run on every tick so we don't need to check inventory in a bunch of places.